contrib/windows/julia-manifest.xml

```diff
@@ -7,6 +7,11 @@
       </requestedPrivileges>
     </security>
   </trustInfo>
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity type='win32'  name='Microsoft.Windows.Common-Controls' version='6.0.0.0'     processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*' />
```

Comment:
I already read the documentation. *It is not required and is meaningless for unsigned assemblies*.